website/content/docs/concepts/acl/auth-methods/oidc.mdx

```diff
@@ -0,0 +1,135 @@
+localhost redirect URI, which is usually
+`http://localhost:4649/oidc/callback`. Logins that use the CLI may
+specify a different host and listening port if needed. A URI with
+this host and port must match one of the configured redirected
+URIs. You must add these same "localhost" URIs to the provider
+as well.
+
+### OIDC Login
+
+#### Nomad UI
+
+1. Select one of the provider links in the Nomad homepage or navigate directly to `/ui/settings/tokens`.
+1. Click one of the buttons for your OIDC auth method of choice.
+1. Complete the authentication with the configured provider.
+
+#### CLI
+
+Execute the `nomad login -method=oidc` command to log in.
+
+```shell-session
+$ nomad login -method=oidc
+
+Complete the login via your OIDC provider. Launching browser to:
+
+    https://myco.auth0.com/authorize?redirect_uri=http%3A%2F%2Flocalhost%3A4649%2Foidc%2Fcallback&client_id=r3qXc2bix9eF...
+```
+
+Your browser opens to the generated URL to complete the provider's login.
+Enter the URL manually if the browser does not automatically open.
+
+The callback listener defaults to `localhost:4649`. You may optionally
+pass a host and port to the callback listener with the
+`-oidc-callback-addr=<host:port>` flag. In this example, you specify custom host and port.
+
+```shell-session
+$ nomad login -method=oidc -oidc-callback-addr=https://custom.host:9080
+```
+
+## OIDC Configuration Troubleshooting
+
+The amount of configuration required for OIDC is relatively small, but it can
+be tricky to debug why things aren't working. The following are tips for setting up OIDC:
+
+- Monitor the log output for the Nomad servers for important
+information about OIDC validation failures.
+
+- Ensure correct redirect URIs in Nmad and on the provider. URIs
+need to match exactly. Check http/https, 127.0.0.1/localhost,
+port numbers, and whether trailing slashes are present.
+
+- The `BoundAudiences` option is typically
+  not required. OIDC providers use the `client_id` as the audience and
+  OIDC validation expects this.
+
+- Check your provider for scopes that are required to receive all of
+  the information you need. You often need to request the scopes
+  `profile` and `groups`, which you may set with
+  `OIDCScopes=["profile", "groups"]` in the auth method configuration.
+
+- If you're seeing claim-related errors in logs, review the provider's docs
+  very carefully to see how they're naming and structuring their claims.
+  Depending on the provider, you may be able to construct a simple `curl`
+  [implicit grant](https://developer.okta.com/blog/2018/05/24/what-is-the-oauth2-implicit-grant-type)
+  request to obtain a JWT that you can inspect. This example decodes the
+  JWT located in the `access_token` field of a JSON response.
+
+      jq --raw-output '.access_token / "." | .[1] | @base64d' jwt.json
+
+- With debug level logging, use the `VerboseLogging` option to log the
+  received OIDC token. This can be helpful when debugging provider setup
```

Comment:
this is such a handy feature. for a second I didn't think it was documented anywhere, but it's over in `api-docs`:

https://nomad-eij8m5abq-hashicorp.vercel.app/nomad/api-docs/acl/auth-methods#verboselogging

I think this page should link there too, not only to the `acl auth-method create` command.

I almost want a `/docs/other-specifications` sub-page to go along with other config file specs, but that may be out of scope here.

but also, in the OIDCScopes bullet, you specify that it's in the auth method config, but here it just says "option". maybe it's clear enough, but I was briefly unsure.